src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/cypress/e2e/ui/newCertifications/newECEAssistantCertification.cy.ts

```diff
@@ -31,18 +31,24 @@ describe("New ECE Assistant Certificate Application", () => {
       beforeEach(() => {
         // Reset cookies and local storage (cy.resetState() is defined in custom commands)
         cy.resetState();
+
         // Set the viewport for the current device.
         cy.viewport(test.width, test.height);
+
         // Visit the base URL (configured in your cypress.config.ts).
         cy.visit("/login");
         cy.document().its("readyState").should("eq", "complete");
+
+        //Reset User State
+        cy.resetUserState();
```

Comment:
shouldn't we reset the UserState before we login? I'm thinking we can move this to line 37. It'll make sure the dashboard is in the correct state.